e2e/test/scenarios/embedding/embedding-dashboard.cy.spec.js

```diff
@@ -222,6 +223,32 @@ describe("scenarios > embedding > dashboard parameters", () => {
       cy.contains(".ScalarValue", "2");
     });
   });
+
+  it("should render error message when `params` is not an object (metabase#14474)", () => {
+    cy.get("@dashboardId").then(dashboardId => {
+      cy.request("PUT", `/api/dashboard/${dashboardId}`, {
```

Comment:
This call is needed to enable embedding + enable filters on those params right?
It looks like it's something we'll do often, should we make a helper function?

Reply:
Maybe. I haven't touched too many embedding tests. So, I'd rather wait and see if this is a pattern we do often. I found we can also do this when we first create a dashboard, so maybe we don't need this extra step at all.

https://github.com/metabase/metabase/blob/5d91e6b7dad73be1a3c8efc3cbbed0c0bafe8bae/e2e/test/scenarios/dashboard-filters/reproductions/29347-remapped-field-values.cy.spec.js#L30-L36

Reply:
This is because some tests don't need the dashboard to have the embedding published. I think we can leave this here as is.

If we have control over the dashboard creation then we would be able to set the embedding to be published since the dashboard creation. Another way is to create another dashboard with embedding published, but that also requires a new API call anyway. So in the end I think reusing the created dashboard is more appropriate.